pipelines/ApPipeMultiTractFakes.yaml

```diff
@@ -0,0 +1,91 @@
+description: AP Pipeline with synthetic/fake sources. Templates are inputs.
+# This pipeline is imported by camera-specific pipelines.
+# You almost certainly want to run one of those, and not this one.
+
```

Comment:
That... is a good question. I think what we really want is everything to be multi-tract in `ApPipe.yaml`, and then this can just _become_ `ApPipeWithFakes.yaml`.